docs/admin/access.rst

```diff
@@ -187,59 +189,60 @@ to a containing project as well.
 An additional step is performed if checking permission for the translation:
 
 
-4. Group :guilabel:`Languages` are matched against accessed translations, it is ignored for component or project level access.
+4. Group :guilabel:`Languages` are matched against accessed translations, it is ignored for component- or project-level access.
 
 .. hint::
 
-   You can use :guilabel:`Language selection` or :guilabel:`Project selection`
+   Use :guilabel:`Language selection` or :guilabel:`Project selection`
    to automate inclusion of all languages or projects.
 
 Checking access to a project
 ++++++++++++++++++++++++++++
 
 A user has to be a member of a group linked to the project or any component
-inside it. Only membership is enough, no specific permissions are needed to
```

Comment:
This sentence is, not good. What is "it"? Relations between groups, links, members and projects/components is somewhat of a blur to me.

Reply:
i rewrote it a bit